index.html

```diff
@@ -7,7 +7,8 @@
   <meta content="width=device-width, initial-scale=1" name="viewport" />
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <meta name="description" content="Automation and Software welcome page.">
-  <meta name="author" content="Claudio Carneiro">
+  <!-- <meta name="author" content="Claudio Carneiro"> -->
```

Comment:
I think both should be kept as authors, don't you think?

Reply:
Is this field "author" used in the GUI?

Reply:
i think both should be kept as authors. this is why I did not delete the original line, just inserted a HTML comment. But although the line itself is not used in GUI (@anacso17), this meta data is indeed passed to the client interface. So maybe I should change the tag `author` to `authors` instead...